### DISCUSSION TITLE
Renovate Usage and License

### DISCUSSION THREAD

**Discussion by hcelaloner** [renovatebot/renovate#17756] [Request Help]
Hi everyone 👋 ,

Forgive my ignorance about open source licenses, I have a simple question about renovate usage and license. Can I freely use renovate to track dependencies across the projects developed in my company? Some of my friends told me that AGPL enforces open source on all components inside. However, if I am not wrong, AGPL enforces open source on all components if we extend renovate and create a public-facing service from it. In a way, prevents launching a competing service on top of it. On the contrary, I solely want to use it to improve developer experience in the company's internal projects.

**Comment by rarkins** [comment]
I am not your lawyer, but as far as I'm aware, your own understanding is completely correct.
